src/card/utils/isValidateCardModule.ts

```diff
@@ -0,0 +1,38 @@
+import { CardBrandName } from '@/card/constants';
+
+const isValidateCardBrand = (brand: string): brand is CardBrandName =>
+  Object.values(CardBrandName).includes(brand as CardBrandName);
+
+const isValidateCardNumber = (cardNumber: string[]) =>
+  cardNumber.length === 4 && cardNumber.every((number) => /^[0-9]{4}$/.test(number));
+
+const isValidateExpirationDate = (expirationDate: string[]) => {
+  if (expirationDate.length !== 2) {
+    return false;
+  }
+  const [month, year] = expirationDate;
+  return /^(0[1-9]|1[0-2]) [0-9]{2}$/.test(`${month} ${year}`);
+};
+
+const isValidateSecurityCode = (securityCode: string[]) => {
+  const securityCodeString = securityCode.join('');
+  if (securityCodeString.length !== 3) {
+    return false;
+  }
+  return /^[0-9]{3}$/.test(securityCodeString);
+};
+
+const isValidatePassword = (password: string[]) => {
+  if (password.length !== 4) {
+    return false;
+  }
+  return password.every((number) => /^[0-9]$/.test(number));
+};
+
+export const isValidateCardModule = {
```

Comment:
```suggestion
export const CardValidationModule = {
```
지금 이름은 함수이름이라서, 이런식으로 표현해야 자연스러울 것 같아요!

Reply:
> 지금 이름은 함수이름이라서, 이런식으로 표현해야 자연스러울 것 같아요!

👍 너무 좋은 피드백이네요!
validate 함수들을 모듈화했다고 하더라도, 네이밍은 컨벤션을 지켰어야하는데 그러지 못했어요.
지금 코드와 같은 함수들을 모듈화한 경우엔, PascalCase를 선호하시나요?

Reply:
저는 보통 PascalCase를 사용하는 편이랍니다 ㅎㅎ